topics/single-cell/tutorials/pseudobulk-analysis/tutorial.md

```diff
@@ -0,0 +1,523 @@
+
+> <agenda-title>Pseudobulk Analysis Pipeline Agenda</agenda-title>
+>
+> 1. Introduction to Pseudobulk Analysis and Data Preprocessing
+>    - Overview of pseudobulk analysis
+>    - Preparing single-cell data for pseudobulk analysis
+>    - Aggregating data into pseudobulk samples
+>    - Inspecting Decoupler outputs
+>
+> 2. Differential Expression Analysis with edgeR
+>    - Preparing input files for edgeR
+>    - Performing differential expression analysis
+>    - Understanding and interpreting the results
+>
+> 3. Visualization and Reporting
+>    - Generating volcano plots for differentially expressed genes
+>    - Summarizing and presenting functional analysis results
+>      
+> 4. Subsetting Samples from the Original AnnData Object
+>    - Extracting AnnData Object with observations of interest 
+>      
+> 5. Key Takeaways and Recommendations
+>    - Reviewing the pseudobulk analysis pipeline
+>    - Suggestions for additional analyses and further exploration
+>
+{: .agenda}
+
+
+# Let's Get the Data!
+
+## Overview of the Data
+
+Our data was extracted from the publication titled _"Elevated Calprotectin and Abnormal Myeloid Cell Subsets Discriminate Severe from Mild COVID-19"_ ({% cite Silvin2020 %}). This dataset was chosen because it was utilized by the developers of the `decoupler` tool for pseudobulk aggregate analysis ({% cite decoupler-pseudobulk %}).
+
+Pseudobulk analysis is an advanced method in single-cell data analysis. For this tutorial, we assume familiarity with common single-cell data formats, such as AnnData or Seurat objects, and experience analysing single-cell data, including clustering and annotating cell types.
+
+If you're new to these concepts, we recommend exploring our other tutorials before performing pseudobulk analysis:
+- [Clustering 3K PBMCs with Scanpy]({% link topics/single-cell/tutorials/scrna-scanpy-pbmc3k/tutorial.md %}): Learn how to cluster and annotate cells in Galaxy using our single-cell tools.
+- [Combining single cell datasets after pre-processing]({% link topics/single-cell/tutorials/scrna-case_alevin-combine-datasets/tutorial.md %}): Understand how to combine multiple datasets into one AnnData object and add metadata from single-cell experiments.
+
+The data object, which you will import from Zenodo into Galaxy via the provided link, has been preprocessed, analysed, and annotated. It includes the following key observations:
+- **cell_type**: The type of cell identified.
+- **disease**: Indicates whether the sample is a control or corresponds to a COVID-19 condition.
+- **individual**: The individual donor or sample identifier.
+- **sex**: The sex of the individual.
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial and name it "Pseudobulk DE Analysis with edgeR"
+> 2. Import the AnnData file from [Zenodo](https://zenodo.org/records/13929549):
```

Comment:
You can use the metadata value here (easier if you ever update zenodo files later, then you only have to change the link in metadata)

```suggestion
> 2. Import the AnnData file from [Zenodo]({{page.zenodo_link}}):
```